python/repositories.bzl

```diff
@@ -265,11 +265,17 @@ def python_register_toolchains(
             platform = platform,
         ))
 
+    # Legacy support for the `_resolved_interpreter` repository name
```

Comment:
we only introduced this very recently. Maybe we should just remove it right now and cut a breaking 0.8.0 rather than carry forward a deprecation and a TODO to eventually remove...

Reply:
+1, I think remove it - _technically_ this is all pre 1.x so we can make breaking changes here

Reply:
This has been removed.